### PR TITLE
[ADD] module base_partner_helper

### PR DESCRIPTION
**Pull request by bealdav**
The purpose of this module is to gather generic partner methods in the same module.
It avoid to grow up excessively the number of module in Odoo for small features

Validation is done in launchpad:
https://code.launchpad.net/~akretion-team/partner-contact-management/7.0-partner-helper-dbl/+merge/221501

### CODE REVIEW

File: base_partner_helper/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    'maintainer': 'Akretion',
+    'category': 'Warehouse',
+    'depends': [
+        'base',
+    ],
+    'description': """
+Partner Helper
+==============
+The purpose of this module is to gather generic partner methods in the same module.
+It avoid to grow up excessively the number of module in Odoo for small features.
```
Comment (pedrobaeza) [inline base_partner_helper/__openerp__.py]: Small typos: 'avoids' and 'modules'

**Comment by pedrobaeza** [comment]
Please fix also PEP8 issues:

```
./base_partner_helper/__openerp__.py:34:80: E501 line too long (83 > 79 characters)
./base_partner_helper/__openerp__.py:35:80: E501 line too long (80 > 79 characters)
./base_partner_helper/partner.py:66:80: E501 line too long (83 > 79 characters)
./base_partner_helper/partner.py:67:1: W391 blank line at end of file
```

**Comment by bealdav** [comment]
Changes are done
Thanks to merge (validated on lp)

File: base_partner_helper/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    'maintainer': 'Akretion',
+    'category': 'Warehouse',
+    'depends': [
+        'base',
+    ],
+    'description': """
+Partner Helper
+==============
+The purpose of this module is to gather generic partner methods.
+It avoid to grow up excessively the number of module in Odoo for small features
```
Comment (pedrobaeza) [inline base_partner_helper/__openerp__.py]: Please fix small typos: "It avoids to grow up excessively the number of modules..."

File: base_partner_helper/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    'maintainer': 'Akretion',
+    'category': 'Warehouse',
+    'depends': [
+        'base',
+    ],
+    'description': """
+Partner Helper
+==============
+The purpose of this module is to gather generic partner methods.
+It avoids to grow up excessively the number of module in Odoo
```
Comment (pedrobaeza) [inline base_partner_helper/__openerp__.py]: Still `module` is in singular, while it must be in plural. Sorry to insist...

Comment (bealdav) [inline base_partner_helper/__openerp__.py]: Don't be sorry to do a real good job, sorry for my part

**Comment by pedrobaeza** [comment]
Thanks for the changes! :+1:

**Comment by pedrobaeza** [comment]
As discused in https://github.com/OCA/partner-contact/pull/15#issuecomment-55425300, maybe it's better to rename it again to _partner_helper_. Sorry for the dizzles, but @hbrunn has convinced me about not using the previously suggested notation with strong arguments.

**Comment by bealdav** [comment]
module rename for the last, I hope ;-)

**Comment by pedrobaeza** [comment]
:+1:
Thank you!

**Comment by bealdav** [comment]
Thanks to you
What are the next steps for the merge ?
What can I do ?
Thanks

**Comment by pedrobaeza** [comment]
In this case, I'm going to take the other approval of @sebastienbeau on LP to proceed with the merge.

Regards.

**Pull request merged**